scripts/fillTestData.js

```diff
@@ -24,6 +26,25 @@ var random = function (max) {
   return Math.floor(Math.random() * max);
 };
 
+// Fisher-Yates shuffle algorith taken from:
+// https://stackoverflow.com/questions/6274339/how-can-i-shuffle-an-array
+/**
+ * Shuffles array in place.
+ * @param {Array} a items An array containing the items.
+ */
+var shuffle = function shuffle(a) {
```

Comment:
We have the same available in `lodash` that we use elsewhere already. Let's switch to using it so that we'll need to carry less code around.

https://lodash.com/docs/4.17.10#shuffle